server/server.py

```diff
@@ -1,25 +1,17 @@
 # server.py
-from flask import Flask, render_template, request
+from flask import Flask, render_template, request, url_for
 import speak
 
-# text = "AI AI minha machadinha !!"
-# lang = "pt"
-
 app = Flask(__name__, static_folder="../static/dist", template_folder="../static")
 @app.route("/", methods=["GET","POST"])
 def index():
     return render_template("index.html")
 
-@app.route("/hello", methods=["GET","POST"])
-def hello():
-    text=''
-    lang=''
+@app.route("/hello/<text>/<lang>")
+def hello(text,lang):
     if request.method == "GET":
-        text = request.form["text"]
-        lang = request.form["lang"]
-        print(text)
         speak.get_speak(text,lang)
-        return flask.url_for ("hello", filename="mp3_path")
+        return flask.url_for("hello", filename="mp3_path")
```

Comment:
Looks like you're still using `flask.url_for(` instead of just `url_for(`. why are you using `"hello"` instead of `"static"` ?

Reply:
Just changed it. :)